docs/source/api/v2/cdns_dnsseckeys_generate.rst

```diff
@@ -13,10 +13,10 @@
 .. limitations under the License.
 ..
 
-.. _to-api-deliveryservices-dnsseckeys-generate:
+.. _to-api-cdns-dnsseckeys-generate:
 
 ****************************************
-``deliveryservices/dnsseckeys/generate``
+``cdns/dnsseckeys/generate``
 ****************************************
```

Comment:
great catch but the length of the surrounding `*` should match the line length of the title

Reply:
Line length matched in aa00b978b0.